truss/remote/baseten/core.py

```diff
@@ -93,23 +95,50 @@ def get_dev_chain_deployment(api: BasetenApi, chain_id: str):
     return newest_draft_deployment
 
 
-def create_chain(
+def create_chain_atomic(
     api: BasetenApi,
-    chain_id: Optional[str],
     chain_name: str,
-    chainlets: List[b10_types.ChainletData],
+    chainlets: List[b10_types.ChainletDataAtomic],
```

Comment:
Why are we not using a datastructre here that has an explicit entrypoint field? So later we don't need to `assert len(entrypoints) == 1`?